### PR TITLE
[INLONG-7400][Sort] Upgrade Oracle CDC to version 2.3

### LINKED ISSUE #7400: [Improve][Sort] Upgrade Oracle CDC to version 2.3.0
### Description

Upgrade Oracle CDC to version 2.3.0.

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by e-mhui**
Upgrade Oracle CDC to version 2.3

- Fixes #7400 

### Motivation

Flink CDC 2.3 has made many improvements to oracle.

### Modifications

1. upgrade Oracle CDC version

```xml
<!--<flink.connector.oracle.cdc.version>2.2.1</flink.connector.oracle.cdc.version>-->
 <flink.connector.oracle.cdc.version>2.3.0</flink.connector.oracle.cdc.version>
```

2. downgrade debezium version

Flink cdc 2.3 depends on the 1.6.4 version of debezium, and the 1.7.0 version of debezium conflicts with it.

```xml
<properties>
    <!-- <debezium.version>1.7.0.Final</debezium.version>-->
     <debezium.version>1.6.4.Final</debezium.version>
 </properties>
```

### CODE REVIEW

**Comment by healchow** [comment]
In this scenario, you can do like this:

1. record your commit hash, it is `3f9f62ccd3ae421f90058ee0da12802f15327772`.
2. `git pull upstream master`
3. `git reset --hard upstream/master`
4. `git cherry-pick 3f9f62ccd3ae421f90058ee0da12802f15327772`
5. `git push your-repo(default is origin) INLONG-7400 (your currently branch) -f`

**Comment by e-mhui** [comment]
> In this PR, it seems that you just need commit the final change, so you can do like this:
> 
> image
> 
> 1. record your commit hash, it is `3f9f62ccd3ae421f90058ee0da12802f15327772`.
> 2. `git pull upstream master`
> 3. `git reset --hard upstream/master`
> 4. `git cherry-pick 3f9f62ccd3ae421f90058ee0da12802f15327772`
> 5. `git push your-repo(default is origin) INLONG-7400 (your currently branch) -f`

Thank you for your help, it has done.

File: inlong-sort/sort-connectors/oracle-cdc/pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <name>Apache InLong - Sort-connector-oracle-cdc</name>
 
     <properties>
-        <debezium.version>1.7.0.Final</debezium.version>
+        <debezium.version>1.6.4.Final</debezium.version>
```
Comment (yunqingmoswu) [inline inlong-sort/sort-connectors/oracle-cdc/pom.xml:33]: The version of oracle-cdc is upgraded but the version of debezium is lower?

**Comment by EMsnap** [comment]
Please add description on the reason for updating version

File: inlong-sort/sort-connectors/oracle-cdc/pom.xml (hunk 1 above)
Comment (e-mhui) [reply inlong-sort/sort-connectors/oracle-cdc/pom.xml:33]: Flink cdc 2.3 depends on the 1.6.4 version of debezium, and the 1.7.0 version of debezium conflicts with it.

**Comment by e-mhui** [comment]
> Please add description on the reason for updating version

It has done.

**Comment by e-mhui** [comment]
> @e-mhui Please update license file of license module

done.

**Review by healchow — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**